ledger/src/blockstore/blockstore_purge.rs

```diff
@@ -1103,33 +1071,28 @@ pub mod tests {
             )
             .unwrap();
 
-        let first_index = {
-            let mut memos_iterator = blockstore
-                .transaction_memos_cf
-                .iterator_cf_raw_key(IteratorMode::Start);
-            memos_iterator.next().unwrap().unwrap().0
-        };
-        let last_index = {
-            let mut memos_iterator = blockstore
-                .transaction_memos_cf
-                .iterator_cf_raw_key(IteratorMode::End);
-            memos_iterator.next().unwrap().unwrap().0
-        };
+        let (first_index, _value) = blockstore
+            .transaction_memos_cf
+            .iterator_cf_raw_key(IteratorMode::Start)
+            .next()
+            .unwrap();
+        let (last_index, _value) = blockstore
+            .transaction_memos_cf
+            .iterator_cf_raw_key(IteratorMode::End)
+            .next()
+            .unwrap();
```

Comment:
We can't use `get_index_bounds()` here since this is operating on a different column.

I could hypothetically parameterize `get_index_bounds()` to take a column instead of Blockstore (currently, `get_index_bounds()` uses `transaction_status_cf`). However, I think that will make things more difficult for a future PR so I'd rather hold off on that for now. IMO, what I did here is still cleaner than what was there (ie no `.0`)